src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java

```diff
@@ -580,15 +578,13 @@ private UpdateOwnershipHoldingsRecordWrapper mapHoldingsRecordWrapper(JsonObject
 
     String sourceHoldingsRecordId = holdingsRecord.getId();
 
-    return new UpdateOwnershipHoldingsRecordWrapper(sourceHoldingsRecordId, holdingsRecord.withId(UUID.randomUUID().toString()));
+    return new UpdateOwnershipHoldingsRecordWrapper(sourceHoldingsRecordId, holdingsRecord.withId(sourceHoldingsRecordId));
```

Comment:
I believe we need to remove the notion of record wrapper, if we are not generating new UUIDs we can just operate with entities on their own